src/service/xhr-impl.js

```diff
@@ -485,7 +489,8 @@ export class FetchResponse {
                'responseType was not "arraybuffer"');
     dev.assert(!this.bodyUsed, 'Body already used');
     this.bodyUsed = true;
-    return Promise.resolve(this.xhr_.response);
+    return /** @type {!Promise<ArrayBuffer>} */ (
```

Comment:
Typically we do errors. But it's fine if it's nullable if it's a valid case. In that case - let's document this well.